### PR TITLE
Additional operators + changes in the boundary stencil coefficients

### PR DESCRIPTION
**Pull request by gdeskos**

### CODE REVIEW

File: src/derive/compute_curvature.cpp [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                        bxhi, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
+                            curvature<StencilIHI>(
+                              i, j, k, idx, idy, idz, field_arr);
+                        });
+                }
+            } // if (!geom.isPeriodic)
+            /* 
+            idim = 1;
+            if (!geom.isPeriodic(idim)) {
+                if (bxi.smallEnd(idim) == domain.smallEnd(idim)) {
```
Comment (michaeljbrazell) [inline src/derive/compute_curvature.cpp:77]: can you finish this or put an abort?

File: unit_tests/derive/test_curvature.cpp [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                curvature.array()(i,j,k)=amr_wind::curvature<amr_wind::StencilInterior>(i,j,k,idx,idy,idz,func.scalar_.array());
+                EXPECT_NEAR(curvature.array()(i,j,k), func.curvature_.array()(i,j,k), tol);
+            }
+        }
+    }
+    
+}
+/*
+TEST_F(CurvatureTest, ihilo)
+{
```
Comment (michaeljbrazell) [inline unit_tests/derive/test_curvature.cpp:38]: why are these commented out?

Comment (gdeskos) [reply unit_tests/derive/test_curvature.cpp:38]: just haven't implemented those yet !

File: src/derive/derive_K.H [hunk 3]
```diff
@@ -10,53 +10,81 @@ namespace amr_wind {
...
 };
 
 struct StencilILO
 {
-    static constexpr amrex::Real c00 =  0.5;
-    static constexpr amrex::Real c01 =  0.5;
-    static constexpr amrex::Real c02 = -1.0;
+    static constexpr amrex::Real c00 =  1./3.;
+    static constexpr amrex::Real c01 =  1.;
+    static constexpr amrex::Real c02 = -4./3.;
```
Comment (sayerhs) [inline src/derive/derive_K.H:46]: So looks like this is going to cause diffs in ABL regression tests. Have you looked at how much of a difference this causes?

File: src/derive/derive_K.H [hunk 4]
```diff
@@ -67,35 +95,67 @@ struct StencilKLO
...
+
+    static constexpr amrex::Real s00 = 8./3.; // axx 
+    static constexpr amrex::Real s01 = -4.; // bxx
+    static constexpr amrex::Real s02 = 4./3.; // cxx
+    static constexpr amrex::Real s10 =  1.; // ayy
+    static constexpr amrex::Real s11 = -2.; // byy
+    static constexpr amrex::Real s12 =  1.; // cyy
+    static constexpr amrex::Real s20 =  s10; // azz
+    static constexpr amrex::Real s21 =  s11; // bzz
+    static constexpr amrex::Real s22 =  s12; // czz
```
Comment (sayerhs) [inline src/derive/derive_K.H:134]: (OCD alert)... can you align these?

File: src/derive/derive_K.H (hunk 3 above)
Comment (michaeljbrazell) [reply src/derive/derive_K.H:46]:
```test 7
    Start  7: abl_godunov

7: Test command: /bin/sh "-c" "/usr/local/bin/mpiexec -n 4  /Users/mbrazell/Codes/amr-wind/build_mac/amr_wind  /Users/mbrazell/Codes/amr-wind/build_mac/test/test_files/abl_godunov/abl_godunov.i time.max_step=10 amr.plot_file=plt amr.checkpoint_files_output=0 amr.plot_files_output=1 amrex.throw_exception=1 amrex.signal_handling=0 > abl_godunov.log && /Users/mbrazell/Codes/amr-wind/build_mac/submods/amrex/Tools/Plotfile/fcompare /Users/mbrazell/Codes/amr-wind/test/AMR-WindGoldFiles/Darwin/AppleClang/11.0.0.11000033/abl_godunov/plt00010 /Users/mbrazell/Codes/amr-wind/build_mac/test/test_files/abl_godunov/plt00010"
7: Test timeout computed to be: 5400
7: 
7:             variable name            absolute error            relative error
7:                                         (||A - B||)         (||A - B||/||A||)
7:  ----------------------------------------------------------------------------
7:  level = 0
7:  density                                          0                         0
7:  gpx                                 1.92758915e-06           0.0001293333767
7:  gpy                                 2.02401295e-06           0.0001291563875
7:  gpz                                5.298073561e-06             1.8539462e-05
7:  mu_turb                              0.02877076275             0.07270538018
7:  p                                  0.0001082316827           1.770245591e-06
7:  temperature                        5.684341886e-13           1.841268755e-15
7:  velocityx                          7.598550437e-05           1.074478421e-05
7:  velocityy                          7.426268384e-05           1.220242844e-05
7:  velocityz                          7.033488545e-06           0.0001706259052
7:  velocity_mueff                       0.02877076275             0.07270354292```

**Comment by michaeljbrazell** [comment]
Let’s wait for a clean nightly test and then merge your stuff. Then we can coordinate with @jrood-nrel to update golds.

**Pull request merged**